core/src/main/scripts/importer/validateData.py

```diff
@@ -3105,6 +3117,24 @@ def main_validate(args):
     collapsing_text_handler.setLevel(output_loglevel)
     logger.addHandler(collapsing_text_handler)
 
+    # set default to unknown because validator can be run independently from cBioPortal
+    cbio_version = "unknown"
+    try:
+        pom_path = get_pom_path()
+
+        # parse xml
+        xml_root = ET.parse(pom_path).getroot()
+        for xml_child in xml_root:
+
+            # to circumvent the default namespace (possibly varying apache url) split on '}'
+            if xml_child.tag.split("}")[1] == "version":
+                cbio_version = xml_child.text
+
+                # output cBioPortal version
+                logger.info("Running validation from cBioPortal version %s" % cbio_version)
+    except Exception:
```

Comment:
This should be more specific, and only catch the exceptions you actually want to ignore by passing through with a default `cbio_version` (I/O errors opening and reading the file). It should not, for instance, ignore manual interrupts by the user and actual runtime errors in logging handlers. If the `ET.parse()` call is the only part that performs file I/O, false positives could be limited even further by putting only that line in the `try` block and moving the handling of the XML contents to [an `else` clause after the `except`](https://docs.python.org/2/reference/compound_stmts.html#index-12).